docker/dockerrun.sh

```diff
@@ -1,2 +1 @@
-docker run -p 9020:9020 -d -v /allen:/allen:rshared danielt/ace2-3d-viewer
-#docker run -p 9020:9020 -d -v /allen:/allen:rshared -e DEPLOYMENT_ENV=dev $docker_image
+docker run --rm -p 9020:80 --name 3d-volume-viewer 3d-volume-viewer-image
```

Comment:
want to keep using /allen as a volume mount, to load onprem Vast files?  Or keep relying on my dev server's config to do that...

Reply:
Leaning towards not, based on a conversation I had with Andrew:
> I am curious if the plan is still to bind-mount /allen into the production container.  That's caused us a lot of problems over the past six years with other containers; this might be a good time to brainstorm alternative approaches to that.

Reply:
I too would lean towards not volume mounting into the prod container. Since you can serve files from a more controlled location on a separate service, this will lend a bit more flexibility and security compared to mounting.